testplan/common/utils/monitor/__init__.py

```diff
@@ -0,0 +1,269 @@
+"""Resource monitor utilities."""
+
+import os
+import uuid
+import time
+import json
+import socket
+import logging
+import threading
+from testplan.common.utils.path import makedirs
+from .collectors import get_collector
+
+
+class EventMonitor(object):
+    """
+    Event monitor will record event start and stop times(seconds)
+    """
+    def __init__(self, parent=None):
+        self.logger = logging.getLogger('Monitor')
```

Comment:
I think in this implementation we may have to use the global TESTPLAN_LOGGER object that repo offers and just register a 'monitor' file handler?
from testplan.logger import TESTPLAN_LOGGER